R-package/R/metrics.R

```diff
@@ -0,0 +1,35 @@
+# [description] List of metrics known to LightGBM. The most up to date list can be found
+#               at https://lightgbm.readthedocs.io/en/latest/Parameters.html#metric-parameters
+#
+# [return] A named logical vector, where each key is a metric name and each value is a boolean.
+#          TRUE if higher values of the metric are desirable, FALSE if lower values are desirable.
+#          Note that only the 'main' metrics are stored here, not aliases, since only the 'main' metrics
+#          are returned from the C++ side. For example, if you use `metric = "mse"` in your code,
+#          the metric name `"l2"` will be returned.
+.METRICS_HIGHER_BETTER <- function() {
+    return(c(
+        "l1" = FALSE
+        , "l2" = FALSE
+        , "mape" = FALSE
+        , "rmse" = FALSE
+        , "quantile" = FALSE
+        , "huber" = FALSE
+        , "fair" = FALSE
+        , "poisson" = FALSE
+        , "gamma" = FALSE
+        , "gamma_deviance" = FALSE
+        , "tweedie" = FALSE
+        , "ndcg" = TRUE
+        , "map" = TRUE
+        , "auc" = TRUE
+        , "binary_logloss" = FALSE
+        , "binary_error" = FALSE
+        , "auc_mu" = TRUE
+        , "multi_logloss" = FALSE
+        , "multiclass" = FALSE
```

Comment:
`multiclass` is alias for `multi_logloss`.

I can confirm that all metric names are listed here. But I don't think that I should review this PR as these changes are all about R-package.

Reply:
oh good eye! I'll remove it

Reply:
removed, thanks!